app/assets/stylesheets/layout_components/comparison-radios.scss

```diff
@@ -37,6 +37,27 @@
   }
 
   &__radio-btn--stacked{
+
+    li{
+      margin-bottom: $base-margin/2;
+    }
```

Comment:
I'm not sure if we normally apply styles directly to tags like this. I figured it would be okay because it is within `&__radio-btn--stacked` but if not I can create a custom class.

Reply:
it should probably follow something like .comparison-radios__list-item? also does it need to have the margin halved? or can it just be the base margin?